src/pages/putaway/list/PutawayListTable.ts

```diff
@@ -37,5 +48,9 @@ class Row extends BasePageModel {
   get statusTag() {
     return this.row.getByTestId('status-0');
   }
+
+  get orderNumber() {
+    return this.row.getByTestId('order-number-0');
```

Comment:
Is the -0 working? The idea of that was that the number at the end is the row number